city_scrapers/spiders/chi_housing.py

```diff
@@ -0,0 +1,79 @@
+from city_scrapers_core.constants import NOT_CLASSIFIED
```

Comment:
I'm guessing this was a from going through the documentation, but it should be removed, along with `Thursday`, `scrapy` and the other `chi_housing` files